docs/dice_verification.md

```diff
@@ -1,40 +1,137 @@
 For example an 'evil maid' attack would be someone accesing your sdcard and change the code to only take in account 5 or 6 dice, so the 24 words would still feel random but the attacker will need to brute force only 5 or 6 dice (very easy).
-This is part of the "do not trust, verify" crypto mottos.
+This is part of the "do not trust, verify" crypto mottos.<br><br>
+**Do NOT use this with any seed you want to use later. This is just for checking that the algorithms get to the same results!!**
+**Or you can download the tools to an airgapped Computer or using tails-OS to check there. Do not use any online computer/tool to do this with real seed phrases!!**
+<br><br>
 
-### Verifying with Ian coleman webpage
+### Dice Rolls / Seed Words example
 
-Go to https://iancoleman.io/bip39 and check show entropy detail:
-<img src="img/dice_entr.png">
+The following 99 dice roll results are used here as an example for a 24 words seed:<br>
+> 655152231316521321611331544441236164664431121534415633526456254462245546236542364246312613322234612
 
-And then make sure to check 'Hex' or 'base 10' (1) and 24 words as mnemonic length (2).
-Do not use 'dice' format because dice 6 will be replaced by 0.
-And then enter the 99 dices numbers in (3) :
-<img src="img/dice_type.png">
+The corresponding 24 seed words are:<br>
+> eyebrow obvious such suggest poet seven breeze blame virtual frown dynamic donor harsh pigeon express broccoli easy apology scatter force recipe shadow claim radio
 
-When the 99 dice number has been entered in the seedsigner and ian coleman page, you will be able to verify that the 24 words are the same.
+<br>
 
+### Creating seed via Dice rolls in Seedsigner (here v0.6.0)
 
-#### Here in real life:
+Power up and boot Seedsigner, go to the 'Tools' menu and select 'New Seed' (with the dice symbols):<br>
+<img src="img/dicedoc/sesi_tools_dice_seed.png" width="600">
 
-We start with first 3 roll, it is important to always read dice roll from left to right to avoid human bias:
-<img src="img/dice_pic1.png">
+Enter the dice numbers one after another on the next screen:<br>
+<img src="img/dicedoc/sesi_dice_1.png" width="600">
 
-Then we arrive at the 99th dice:
-<img src="img/dice_pic2.png">
+Go on until the end (99 dice roll numbers):<br>
+<img src="img/dicedoc/sesi_dice_2.png" width="600">
 
-Then same 24 words! :
-<img src="img/dice_pic3.png">
+After that the 24 seed words are shown (in 6 screens of 4 words each):<br>
+<img src="img/dicedoc/sesi_seed_1.png" width="600">
+<br>**.....**<br>
+<img src="img/dicedoc/sesi_seed_2.png" width="600">
 
-Now we are sure that the dice derivation is correct and we can unplug everything and do a new dice roll only on the seedsigner.
+The fingerprint for this seed is:<br>
+<img src="img/dicedoc/sesi_finger_print.png" width="600">
+<br><br>
+
+### Create new wallet from seed in Sparrow Wallet to see xpub/zpub and addresses
```

Comment:
Please consider to add some more whitespace and describe the purpose of this next step.  It seemed as if there could be some pause/ breathing space after creating the dice seed in the Seedsigner, and _then_ move onto the next steps. 

Perhaps: "Having now created a dice-based seed in the SeedSigner, we will now compare those details to what appears in 
- Sparrow desktop software, 
- Ian Coleman website
- Seed Guide website

Simliarly, please look to add some vertical white space when swapping to the next websites